products/compute/api.yaml

```diff
@@ -1763,11 +1763,7 @@ objects:
           - :SCTP
           - :ICMP
       - !ruby/object:Api::Type::ResourceRef
-        # TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/200)
-        # After adding the RegionBackendService resource, move this field out
-        # of the common fields for forwarding rules. This field should refer
-        # to a RegionBackendService for a ForwardingRule and to a BackendService
-        # for a GlobalForwardingRule.
+        # TODO(RegionBackendService)
```

Comment:
It isn't clear what we need to do to resolve this TODO

Reply:
You know, I could fix it. Or, I could delete it, since I think you CAN use Global backend services with regional Forwarding Rules. Unclear! Hold off on reviewing until I ping you?